include/he.h

```diff
@@ -335,13 +335,35 @@ typedef he_return_code_t (*he_inside_write_cb_t)(he_conn_t *conn, uint8_t *packe
  * @param length The length of the entire packet in bytes
  * @param context A pointer to the user defined context
  * @see he_conn_set_context Sets the value of the context pointer
+ * @deprecated Use he_outside_write_ex_cb_t instead.
  *
  * Whenever Helium needs to do an outside write this function will be called. On Linux this would
  * usually be writing to a UDP socket to send encrypted data over the Internet.
  */
 typedef he_return_code_t (*he_outside_write_cb_t)(he_conn_t *conn, uint8_t *packet, size_t length,
                                                   void *context);
 
+/// Instruct the outside_write_cb function to set the DF flag on the IP packet
+#define HE_OUTSIDE_WRITE_DONT_FRAGMENT 0x0001
```

Comment:
Can we use `0x00000001` to denote uint32_t ?

Reply:
Done.